.github/workflows/deploy.yml

```diff
@@ -34,32 +38,11 @@ jobs:
       - name: Run unit tests
         run: npm run test:ci
 
-      - name: Configure AWS credentials
-        uses: aws-actions/configure-aws-credentials@v1
-        with:
-          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
-          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
-          aws-region: eu-west-2
-
-      - name: Log in to Amazon ECR
-        id: login-ecr
-        uses: aws-actions/amazon-ecr-login@v1
-
-      - name: Docker Build, tag and push image to Amazon ECR
-        env:
-          ECR_REGISTRY: ${{ steps.login-ecr.outputs.registry }}
-          ECR_REPOSITORY: tis-revalidation-v2
-          IMAGE_TAG: ${{ github.sha }}
-        run: |
-          docker build -t $ECR_REGISTRY/$ECR_REPOSITORY:$IMAGE_TAG .
-          docker push $ECR_REGISTRY/$ECR_REPOSITORY:$IMAGE_TAG
-          docker tag $ECR_REGISTRY/$ECR_REPOSITORY:$IMAGE_TAG $ECR_REGISTRY/$ECR_REPOSITORY:latest
-          docker push $ECR_REGISTRY/$ECR_REPOSITORY:latest
-
```

Comment:
These steps still need to be done when deploying.

Reply:
these are the steps done on push to master only

Reply:
line 45 ` if: github.ref == 'refs/heads/master' ` you gave me the syntax to run on master branch

Reply:
But you haven't added those steps in to the the `deploy` job, so even on master the docker image won't be built anymore.

Reply:
ohhhh snaap oops

Reply:
steps added